labs-ios-starter/Surveys/View Controllers/TopicViewController.swift

```diff
@@ -110,6 +113,8 @@ class TopicViewController: LoginViewController, NSFetchedResultsControllerDelega
                         self.spinner.stopAnimating()
                         topicsCollectionView.reloadData()
                     } catch {
+                        self.spinner.stopAnimating()
+                        topicsCollectionView.reloadData()
```

Comment:
good catch. pun intended

Reply:
thx... we could probably use `defer { }` here, but that always worries me lol